### PR TITLE
[Splash Screen] Skip image/video using keyboard/mouse

### PR DESCRIPTION
**Pull request by veksha**
I have verified that:
- [x] These changes work, by building the extension and testing.
- [x] That the changes comply with the [rules](https://github.com/darklinkpower/PlayniteExtensionsCollection#contributing) indicated in the repository.
- [x] Pull request is targeting `master` branch.

resolves https://github.com/darklinkpower/PlayniteExtensionsCollection/issues/503

### CODE REVIEW

**Comment by veksha** [comment]
also it seems that this PR fixes the following issue too:

> AleNaz: im using xenia with splash screen
> AleNaz: and it doesnt focus xenia
> AleNaz: so i have to alt tab to xenia

File: source/Generic/SplashScreen/SplashScreen.cs [hunk 1]
```diff
@@ -257,6 +283,7 @@ private void CreateSplashVideoWindow(bool showSplashImage, string videoPath, str
                 currentSplashWindow.Content = new SplashScreenImage { DataContext = new SplashScreenImageViewModel(generalSplashSettings, splashImagePath, logoPath) };
                 PlayniteApi.Dialogs.ActivateGlobalProgress((a) =>
                 {
+                    ActivateSplashWindow(currentSplashWindow);
```
Comment (darklinkpower) [inline source/Generic/SplashScreen/SplashScreen.cs:286]: Just wondering, why is this method being called twice? Also why aren't the `Activate` and `CaptureMouse` methods being called (or replacing) where the `Show` method is being used above?

Comment (veksha) [reply source/Generic/SplashScreen/SplashScreen.cs:286]: `ActivateGlobalProgress` activates progress window. we must activate splash window **after** progress window is shown. it seems that we can do it only inside anonymous function that is passed to `ActivateGlobalProgress` call.
and `ActivateGlobalProgress` is called **twice**, for video and then for splash after video.

File: source/Generic/SplashScreen/SplashScreen.cs [hunk 2]
```diff
@@ -228,12 +240,25 @@ private void CreateSplashVideoWindow(bool showSplashImage, string videoPath, str
...
+                {
+                    videoWaitHandle.Set();
+                }
+                else
+                {
+                    currentSplashWindow.Close();
+                }
+            };
+            currentSplashWindow.KeyDown += SkipVideoSplash;
+            currentSplashWindow.MouseDown += SkipVideoSplash;
```
[the remark this answers is not in the document]
Comment (veksha) [reply source/Generic/SplashScreen/SplashScreen.cs]: done

File: source/Generic/SplashScreen/SplashScreen.cs [hunk 3]
```diff
@@ -299,16 +326,22 @@ private void CreateSplashImageWindow(string splashImagePath, string logoPath, Ge
...
                 // Window is set to topmost to make sure another window won't show over it
                 Topmost = true
             };
 
+            void SkipImageSplash(object sender, EventArgs e)
+            {
+                currentSplashWindow.Close();
+            };
+            currentSplashWindow.KeyDown += SkipImageSplash;
+            currentSplashWindow.MouseDown += SkipImageSplash;
```
[the remark this answers is not in the document]
Comment (veksha) [reply source/Generic/SplashScreen/SplashScreen.cs]: done. combined to common event SkipSplash.

**Pull request merged**